### PR TITLE
view modal for specific event  

### LINKED ISSUE #129: Frontend: Modal View for Events Detail Visualization for Admins
When I am logged in as an admin, I want to be able to see the detail about a specific event so I can decide need to do something about it.

Create a Modal View Events Detail Visualization for Admins like shown below

image

Note: Create the components with the appropriate style. Please check the wireframes https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by MaherHussain**
# Description

 this PR is about for view modal for specific event  , which show event details to admin  .

Fixes #129

# How to test?

tested in storybook and works fin 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/Admin-Event-details/Admin-Event-modal-view.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      topic: 'Topic: JavaScript week3 homework,classes,Function',
+      code: 'maherweb.dk',
+    },
+    {
+      studentName: 'DemoStudent3',
+      class: 'class 15',
+      topic: 'Topic: JavaScript week3 homework,classes,Function',
+      code: 'maherweb.dk',
+    },
+  ];
```
Comment (tpetrina) [inline src/client/containers/Admin-Event-details/Admin-Event-modal-view.js:29]: Move this to props and then only use this list of students in Storybook.

**Review by tpetrina — CHANGES_REQUESTED** [review]

**Review by Yaser-Aldammad — DISMISSED** [review]
good job

File: src/client/containers/Admin-Event-details/Admin-Modal-View.stories.js [hunk 2]
```diff
@@ -1,13 +1,52 @@
...
+  return (
+    <AdminModalView
+      eventTitle="Elastic - Week 46"
+      eventDate="Wednesday | 11 Nov 20|10:00 CEST"
+      mentor="madsssssssssssssssssssssssssssssssssssssssssss"
+      eventLink="maherweb.dk"
+      items={items}
+    />
+  );
+}
```
Comment (tpetrina) [inline src/client/containers/Admin-Event-details/Admin-Modal-View.stories.js:36]: Why do you have this export in stories?

**Review by tpetrina — CHANGES_REQUESTED** [review]

**Review by tpetrina — APPROVED** [review]

**Review by Elia-Git — APPROVED** [review]
Good job mate!

**Pull request merged**